clustering/jgroups/extension/src/test/resources/org/jboss/as/clustering/jgroups/subsystem/subsystem-jgroups-transform-reject.xml

```diff
@@ -75,12 +76,7 @@
             </relay>
         </stack>
         <stack name="minimal">
-            <transport type="UDP">
-                <default-thread-pool min-threads="11"/>
-                <internal-thread-pool queue-length="22"/>
-                <oob-thread-pool keepalive-time="34"/>
-                <timer-thread-pool max-threads="43"/>
```

Comment:
This kind of thing always makes me nervous in a transformer test; i.e. that the thing being removed may be the thing that needed transformation.

A simple "It's not" reply will make me calm. ;)